t/lib/ext-plugin.lua

```diff
@@ -169,6 +169,8 @@ function _M.go(case)
             local entry = call_req:Args(1)
             assert(entry:Name() == "x")
             assert(entry:Value() == "z")
+        elseif case.get_request_body then
+            assert(call_req:Method() == a6_method.POST)
```

Comment:
Do we need to check the request_body's value?

Reply:
I want to validate different content-type, so the value of request body is not fixed.

Reply:
actions.result would check the value in request body?